src/seedu/addressbook/commands/FindCommand.java

```diff
@@ -48,13 +44,31 @@ public CommandResult execute() {
      */
     private List<ReadOnlyPerson> getPersonsWithNameContainingAnyKeyword(Set<String> keywords) {
         final List<ReadOnlyPerson> matchedPersons = new ArrayList<>();
+        List<String> nameSplitByWhiteSpace = new ArrayList<>();
+        ArrayList<String> lowerCaseNames = new ArrayList<>();
+        Set<String> loweredKeywords = new HashSet<>();
         for (ReadOnlyPerson person : addressBook.getAllPersons()) {
-            final Set<String> wordsInName = new HashSet<>(person.getName().getWordsInName());
-            if (!Collections.disjoint(wordsInName, keywords)) {
+            nameSplitByWhiteSpace = person.getName().getWordsInName();
+            for (String split : nameSplitByWhiteSpace) {
+                lowerCaseNames.add(makeLowerCase(split));
+            }
+            final Set<String> wordsInName = new HashSet<>(lowerCaseNames);
```

Comment:
Is this variable being used? Unused variables should be removed from the code as it improves readability.